docs/guides/chip_tool_guide.md

```diff
@@ -668,6 +674,57 @@ The following flags are available:
     Here, _<onoff\>_ is a `[0/1]` flag, which when set to `1` prints the trace
     data with automation logs to the console.
 
+<hr>
+
+### Commissioner name and ID flags
+
+All CHIP Tool commands can be used together with the following
+commissioner-related flags:
+
+-   `--commissioner-name`
+-   `--commissioner-nodeid`
+
+These flags let you inform the Matter device about which commissioner it must
+interact with. They are especially useful in scenarios where you are working
+with more than one fabric, but you can also use it with a single commissioner.
+
+#### `--commissioner-name` flag
+
+As per the CHIP Tool implementation, the commissioners are required to have the
+following names: `alpha` for the first one, `beta` for the second one, `gamma`
+for the third one, `4` for the fourth one, `5` for the fifth one, and so on.
+
+If you don't use this flag, the CHIP Tool assumes that the command is meant for
+the `alpha` commissioner.
+
+**Example of commands:**
+
+```
+$ ./chip-tool any subscribe-by-id '0x0028,0x0028,0x0101,0x0028,0x0028,0x0028' '5,6,0,1,2,4' 100 1000 1 '0,0,1,0,0,0' --keepSubscriptions true
+$ ./chip-tool any subscribe-by-id '0x0028,0x0028,0x0101,0x0028,0x0028,0x0028' '5,6,0,1,2,4' 100 1000 2 '0,0,1,0,0,0' --keepSubscriptions true --commissioner-name beta
+$ ./chip-tool any subscribe-by-id '0x0028,0x0028,0x0101,0x0028,0x0028,0x0028' '5,6,0,1,2,4' 100 1000 3 '0,0,1,0,0,0' --keepSubscriptions true --commissioner-name gamma
+```
+
+#### `--commissioner-nodeid` flag
+
+Each commissioner has a node ID that you can use instead of its name when
+providing the commissioning flags.
```

Comment:
That's not what the flag does.  What the flag does is that once you have selected a fabric (via --commissioner-name), you can tell chip-tool what node id to use on that fabric via --commissioner-nodeid.  The default behavior if this flag is not specified is to use whatever value is in storage (probably none, unless you set up the storage by hand to have a value), falling back to decimal "112233".

Note that the help text for this flag is correct about its behavior....